msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md

```diff
@@ -59,7 +59,29 @@ IPublicClientApplication app =
     .WithBroker(options)
     .Build();
 
-var authResult = await app.AcquireTokenInteractive(new List<string>() { "User.Read" }).ExecuteAsync();
+IAccount existingAccount = await FetchExistingAccountFromCache().ConfigureAwait(false);
```

Comment:
@bgavrilMS we likely need to document what `FetchExistingAccountFromCache` - it's referenced in the code snippet but not clear what the customer needs to do here or where the function implementation lives.

Reply:
maybe we can add the getAccounts API and also how to get the OS Accounts, thought we already had it

Reply:
nm, I see we are dropping it